javascripts/footer.js

```diff
@@ -1,6 +1,11 @@
 $(function() {
   $(".collapsible-section").on("click", function(){
-    $(this).toggleClass("collapsed");
+    var windowsize = $window.width();
+    if (windowsize <= 1170) {
+      $(this).toggleClass("collapsed");
```

Comment:
Really not sure if we want this behavior or not. Do we want to collapse the section once they've clicked on a link?